ui/components/component-library/avatar-base/avatar-base.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import './index.scss';
+
+function AvatarBase(props) {
+  const { size } = props;
+  return <div className={classnames('avatar-base', ` --size-${size}`)}></div>;
+}
+
+AvatarBase.propTypes = {
+  size: PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl']),
```

Comment:
All of our components should have proptype descriptions!\

Reply:
Added